grakn-kb/src/main/java/ai/grakn/factory/GraknSessionImpl.java

```diff
@@ -114,7 +116,7 @@ private static Properties getTxRemoteProperties(String engineUrl, Keyspace keysp
     private static Properties getTxInMemoryProperties(){
         Properties inMemoryProperties = new Properties();
         inMemoryProperties.put(GraknConfigKey.SHARDING_THRESHOLD, 100_000);
-        inMemoryProperties.put(GraknTxAbstract.NORMAL_CACHE_TIMEOUT_MS, 30_000);
+        inMemoryProperties.put(GraknConfigKey.SESSION_CACHE_TIMEOUT, 30_000);
         inMemoryProperties.put(FactoryBuilder.KB_MODE, TxFactoryTinker.class.getName());
```

Comment:
I'm wondering if this code technically works... I suspect you actually want to call
```
inMemoryProperties.setProperty(GraknConfigKey.SESSION_CACHE_TIMEOUT.name(), 30_000);
```
otherwise, the key is going to be a `GraknConfigKey`, when it should be a string.

Reply:
That's likely true for the sharding threshold as well. I suppose this never got caught because the In memory graph is only ever used as a toy